serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc

```diff
@@ -113,6 +113,41 @@ In the previous example, Kogito uses `stock-service_API` as configuration key.
 A Kubernetes service endpoint can be used as a service URL if the target service is within the same cluster, such as `http://myservice.mynamespace.cluster.svc.local`.
 ====
 
+=== Using URI alias
+
+As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as configuration key and in function definitions. 
```

Comment:
```suggestion
As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as a configuration key in function definitions. 
```

Reply:
This change the meaning, I meant you can use the alias as configuration AND also use the alias in function definitions

Reply:
Ah, okay. Done, I've updated the suggestion